gpkit/constraints/set.py

```diff
@@ -262,3 +256,36 @@ def process_result(self, result):
         for constraint in self:
             if hasattr(constraint, "process_result"):
                 constraint.process_result(result)
+
+
+def raise_badelement(cns, i, constraint):
+    "Identify the bad element and raise a ValueError"
+    cause = "" if not isinstance(constraint, bool) else (
+        "Did the constraint list contain"
+        " an accidental equality?")
+    if len(cns) == 1:
+        loc = "as the only constraint"
+    elif i == 0:
+        loc = "at the start, before %s" % cns[i+1]
+    elif i == len(cns) - 1:
+        loc = "at the end, after %s" % cns[i-1]
+    else:
+        loc = "between %s and %s" % (cns[i-1], cns[i+1])
+    raise ValueError("%s was found %s. %s"
```

Comment:
SUUUUPER nitpicky but why not put a space before "Did" in the `cause` string and then make this `raise ValueError("%s was found %s.%s"`